crates/e2e/macro/src/config.rs

```diff
@@ -28,6 +29,8 @@ pub struct E2EConfig {
     whitelisted_attributes: WhitelistedAttributes,
     /// Additional contracts that have to be built before executing the test.
     additional_contracts: Vec<String>,
+    /// Custom environment for the contracts, if specified. Otherwise `DefaultEnvironment` is used.
+    environment: Option<Path>,
```

Comment:
I'd generally keep these as `syn::Path` to not have this confused with `std::path::Path` for example. It also matches the style of surrounding code

Reply:
:white_check_mark: